mempool/src/core_mempool/mempool.rs

```diff
@@ -189,6 +202,13 @@ impl Mempool {
                 }
                 total_bytes += txn_size;
                 block.push(txn);
+                if let Some(ranking_score) = self.transactions.get_ranking_score(&address, seq) {
```

Comment:
Can you just use `txn.ranking_score` here to avoid additional hashmap look up?

Reply:
Unfortunately, not directly. Only MempoolTransaction has that info and only SignedTransactions is returned.